cwebp.c:186:47: error: no member named 'cross_color_transform_bits' in 'struct WebPAuxStats'

Index: examples/cwebp.c
--- examples/cwebp.c.orig
+++ examples/cwebp.c
@@ -182,9 +182,9 @@ static void PrintFullLosslessInfo(const WebPAuxStats* 
   if (stats->lossless_features & 1) {
     fprintf(stderr, " prediction=%d", stats->transform_bits);
   }
-  if (stats->lossless_features & 2) {
-    fprintf(stderr, " cross-color=%d", stats->cross_color_transform_bits);
-  }
+  //if (stats->lossless_features & 2) {
+  //  fprintf(stderr, " cross-color=%d", stats->cross_color_transform_bits);
+  //}
   fprintf(stderr, " cache=%d\n", stats->cache_bits);
   if (stats->palette_size > 0) {
     fprintf(stderr, "  * Palette size:   %d\n", stats->palette_size);
